src/components/home/home-page.vue

```diff
@@ -1,35 +1,31 @@
 <script setup lang="ts">
 import { RouterLink } from 'vue-router';
-const Name = 'Frances Gail T. Quipit'
-const currentDate: string = new Date().toLocaleDateString();
+const Name = 'Frances Gail T. Quipit';
 </script>
 
 <template>
-        <div class="center">
-            <h2>{{ Name }}</h2>
-        <br>
-            {{ currentDate }}
-        </div>
-        <div>
-            <RouterLink to="/todo">
-                Go to app
-            </RouterLink>
-        </div>
+  <div class="center">
+    <h2>{{ Name }}</h2>
+    <br />
+    1/16/2025
+  </div>
+  <div>
+    <RouterLink :to="{ name: 'todo' }"> Go to app </RouterLink>
```

Comment:
use enum/constants for Route names
```suggestion
    <RouterLink :to="{ name: RouteName.TODO }"> Go to app </RouterLink>
```

Reply:
done